test/utils/mockutils/mockcond/body.go

```diff
@@ -0,0 +1,46 @@
+package mockcond
+
+import (
+	"bytes"
+	"io"
+	"net/http"
+	"strings"
+)
+
+// Body returns a check expecting body to match template text.
+func Body(expectedBody string) Check {
```

Comment:
Cool stuff! I wonder if we can rename it to be more like primitives that we already are familiar with - 
```
If: And{ 
  IfSuffix(...),
  IfBody(...),
},
// Evaluation success
Then: {
   SendResponse(...),
},
// Evaluation failure
Else: {
}
```
```